### PR TITLE
[nl interface] update the feedback form being used

### PR DESCRIPTION
**Pull request by chejennifer**
update nl interface to use this shorter feedback form instead that just asks user for their feedback: https://forms.gle/ADxVJzmjid3ynBo26

### CODE REVIEW

File: static/js/utils/nl_interface_utils.ts [hunk 1]
```diff
@@ -19,14 +19,14 @@
  */
 
 const FEEDBACK_LINK =
-  "https://docs.google.com/forms/d/e/1FAIpQLSfqndIayVhN1bN5oeZT0Te-MhhBMBR1hn97Lgr77QTOpga8Iw/viewform?usp=pp_url";
+  "https://docs.google.com/forms/d/e/1FAIpQLSe9SG0hOfrK7UBiOkQbK0ieC0yP5v-8gnQKU3mSIyzqdv6WaQ/viewform?usp=pp_url";
```
Comment (pradh) [inline static/js/utils/nl_interface_utils.ts:22]: 1) Will "What was the query" get auto-filled?  If so, move it after the "Thank you!".

2) Email - should we make it optional?

3) In the Feedback textbox, should we add a few words of text?  Like the Bard form.... concretely:

```
How would you rate the overall relevance of the charts to your question? 
- Not at all relevant
- Somewhat relevant
- Very relevant

If you have anything else to share on what we could fix or improve, please let us know.
[........]
```

**Review by pradh — COMMENTED** [review]
Thanks!

File: static/js/utils/nl_interface_utils.ts (hunk 1 above)
Comment (chejennifer) [reply static/js/utils/nl_interface_utils.ts:22]: updated the form, let me know what you think: https://forms.gle/1ih4oPmKg2Q8Mg6A8

Comment (pradh) [reply static/js/utils/nl_interface_utils.ts:22]: Thanks!

Make "If you have anything else to share on what we could fix or improve, please let us know." optional and should we still keep the email, but optional as 3rd question?

**Review by pradh — APPROVED** [review]

**Review by dwnoble — APPROVED** [review]
Thank you!

File: static/js/utils/nl_interface_utils.ts (hunk 1 above)
Comment (chejennifer) [reply static/js/utils/nl_interface_utils.ts:22]: done

**Pull request merged**